libs/sdk-core/src/models.rs

```diff
@@ -1169,32 +1169,53 @@ impl TryFrom<i32> for SwapStatus {
 /// saved to the persistent storage.
 ///
 /// The SwapInfo has a status which changes accordingly, documented in [SwapStatus].
+///
+/// Static fields are those which don't change for the lifetime of a swap, whereas dynamic fields may change.
+
 #[derive(Clone, PartialEq, Eq, Debug, Serialize, Deserialize)]
 pub struct SwapInfo {
-    //static immutable data
+    /// Bitcoin address for this swap. Sats sent to this address will be swapped. Static field.
     pub bitcoin_address: String,
+    /// Relative time lock start, received from [SwapperAPI::create_swap]. Static field.
     pub created_at: i64,
+    /// Relative time lock for the timeout for the script to be redeemed before swap fails. Static field.
     pub lock_height: i64,
+    /// sha256 hash of preimage to used in the claim sript. Static field.
     pub payment_hash: Vec<u8>,
+    /// Secret to claim the swap. Static field.
     pub preimage: Vec<u8>,
+    /// Secret claim key for the bitcoin address. Static field.
     pub private_key: Vec<u8>,
+    /// Public key in binary format of the private claim private key. Static field.
     pub public_key: Vec<u8>,
+    /// The public key in binary format from the swapping service. Received from [SwapperAPI::create_swap]. Static field.
     pub swapper_public_key: Vec<u8>,
+    /// The lockingsscript for the generated bitcoin address. Received from [SwapperAPI::create_swap]. Static field.
     pub script: Vec<u8>,
 
-    // dynamic data
+    /// bolt11 invoice to claim the sent funds. Dynamic field.
     pub bolt11: Option<String>,
+    /// Amount of sats payed to the claim address. Dynamic field
```

Comment:
Same here for Dynamic field. I would remove this, too low level and implementation details.

Reply:
Done